drivers/nfs.py

```diff
@@ -218,11 +213,61 @@ def scan_exports(target):
         entry.appendChild(subentry)
         textnode = dom.createTextNode(access)
         subentry.appendChild(textnode)
+    return dom
+
+
+def _scan_exports_nfs4_only(target, transport, dom, element):
```

Comment:
given that we have `_scan_exports_nfs3` this should probably just be `_scan_exports_nfs4` for consistency.

Reply:
I hesitated to add the `_only` or not, both are okay for me so I can remove it if you prefer.

Reply:
done :)